src/main/java/org/jenkinsci/plugins/workflow/cps/CpsWhitelist.java

```diff
@@ -103,6 +104,9 @@ public boolean permitsStaticMethod(Method method, Object[] args) {
             String driverTo = "org.codehaus.groovy.runtime.";
             if (cn.startsWith(driverFrom)) {
                 try {
+                    if (Modifier.isPrivate(method.getModifiers())) {
+                        return true;
```

Comment:
Sounds like a good idea to me. I can imagine some other cases where we may want to explicitly write CPS code, which would benefit from that as well.